mage_ai/data_preparation/models/pipelines/integration_pipeline.py

```diff
@@ -39,6 +39,10 @@ def data_loader(self) -> Block:
     def data_exporter(self) -> Block:
         return find(lambda x: BlockType.DATA_EXPORTER == x.type, self.blocks)
 
+    @property
+    def transformer(self) -> Block:
+        return find(lambda x: BlockType.TRANSFORMER == x.type, self.blocks)
+
```

Comment:
i remember xiaoyou mentioning that some users wanted to have multiple transformers in a data integration pipeline. not sure if that's being prioritized but it might break this in the future